src/test/java/org/truffleruby/tck/RubyDebugTest.java

```diff
@@ -200,6 +234,29 @@ private void stepInto(final int size) {
         run.addLast(() -> suspendedEvent.prepareStepInto(size));
     }
 
+    private void assertArguments(final int line, final String code, final String... expectedArgs) {
```

Comment:
Is this just dead code? it looks like all the callers are commented out.

Reply:
Fixing.